text/docs-decision-tree.md

```diff
@@ -0,0 +1,46 @@
+
+This has led to one change and necessitates a new one!:
+
+1.  @jlengstorf already created the [@gatsbyjs/docs team](https://github.com/orgs/gatsbyjs/teams/docs) in GitHub that automatically notifies whomever is on the team whenever a PR includes edits to anything within /docs/
+
+2.  New change needed: I've written up this RFC about a docs decision tree so that when the @gatsbyjs/docs team is notified, they can make decisions in a unified manner according to a set of agreed-upon principles (which can be adjusted over time). This will help the OSS process escape any biases and fickleness. This will be tested over Hacktoberfest and facilitate discussion through comments on this RFC.
+
+## Why is this important?
+
+In order to be clear and professional, it's important for the docs to be consistent in style, tone, and purpose, and contributors also ought to be able to depend upon a predictable contribution process. This decision making process will help us act as a unified team according to a set of agreed-upon principles (which can be adjusted over time) and help us escape any biases and fickleness as we make the docs more awesome.
+
+## What is required to make this happen?
+
+- [ ] Get enough feedback and testing on the [docs decision tree](https://whimsical.co/78PmoqFTbJJxpXHA1a6gba) with Hacktoberfest to polish the idea and this RFC
+- [ ] Merge the RFC
+- [ ] Regularly revisit the RFC with the @gatsbyjs/docs team to revise it
+- [ ] Blog about the decision-making process and refer to the blogpost in difficult situations to help contributors understand our decision-making process
+
+## Who will own this?
+
+Learning AoR: Shannon Soper
+
+## What are the drawbacks and/or downsides of doing this?
+
+Creating a decision tree can sometimes mean that we could be tempted to blindly follow it and be too strict, thinking of it as a set of hard rules rather than guiding principles. We could also swing too far in the other direction and not use it enough and be inconsistent with contributors and thereby create inconsistent docs.
+
+## Are there alternative approaches that would work to accomplish the same goal?
+
+I can't see another way to accomplish the goal of becoming a unified, unbiased (as far as it is possible) @gatsbyjs/docs team. There are other formats (i.e. we could print the docs decision tree and laminate it, for example, or make a rap about it), but the core heuristic ought to exist in some form or another.
+
+## How will we drive adoption?
+
+Test it during Hacktoberfest. For the time being, that's good enough because @amberley and I are already getting tagged as @docsteam, so we can test the decision tree anytime we get tagged.
```

Comment:
Any feedback from Hacktoberfest now that it's been over for a while?

Reply:
I would say the things that went well:
- the code-owners @gatsbyjs/docs team did a great job gathering all docs-related issues into one spot since it tagged the team automatically for anything in /docs/ /tutorial/ or 'doc-links.yml' or 'tutorial-links.yml'
- The OSS team has done a good job merging things that don't need my specialization - for example, without waiting for docs team approval, they merge stuff that makes docs more accurate, more up-to-date, more in line with the style guide. I like this. Don't know if they like it...:)

The things that are still remaining issues imo:
- I don't know how to change code owners notifications and don't know who owns it or how to look at it, so I'm not sure if my request that I only be notified about changes to 'doc-links.yml' and 'tutorial-links.yml' instead of all docs changes has been accomplished
- I also wanted to be notified about any docs being deleted or added to /docs/, not sure if it's possible to automate this.
- Sometimes, really big changes to the docs need a technical review after I do a docs review, and I don't know who to assign or ping.